becquerel/core/spectrum.py

```diff
@@ -123,27 +124,29 @@ def __init__(self, counts=None, cps=None, uncs=None, bin_edges_kev=None,
 
         if not (counts is None) ^ (cps is None):
             raise SpectrumError('Must specify one of counts or CPS')
+
         if counts is not None:
             if len(counts) == 0:
                 raise SpectrumError('Empty spectrum counts')
+            if uncs is None and np.any(np.array(counts) < 0):
```

Comment:
Is there a more elegant way than converting counts to a `numpy` array?

Reply:
Counts needs to be converted to an array later anyways so I don’t think this is an issue. We should probably cast the counts array with np.asarray